apps/mobile/src/components/Typography.tsx

```diff
@@ -21,7 +21,7 @@ type TypographyProps = PropsWithChildren<{
       };
 }>;
 
-export function Typography({ font, className, style, children }: TypographyProps) {
+export function Typography({ font, style, children }: TypographyProps) {
```

Comment:
Passing the `className` is only important for type safety, we don't actually need to pass the string in, `nativewind` just passes things through via the `style` prop.